app/src/main/java/com/rkm/tasky/feature/agenda/screen/DateHelper.kt

```diff
@@ -0,0 +1,68 @@
+package com.rkm.tasky.feature.agenda.screen
+
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import kotlinx.datetime.Clock
+import kotlinx.datetime.DateTimeUnit
+import kotlinx.datetime.Instant
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.atStartOfDayIn
+import kotlinx.datetime.plus
+import kotlinx.datetime.toLocalDateTime
+import kotlinx.datetime.todayIn
+import javax.inject.Inject
+
+@Module
+@InstallIn(SingletonComponent::class)
+object DateHelperModule {
+    @Provides
+    fun providesDateHelper(): DateHelper {
+        return DateHelper()
+    }
+}
+
+class DateHelper @Inject constructor() {
```

Comment:
Try to keep DI modules separate from your production code. It helps to have them in a central place for more efficient maintainability of the project instead of having to look through the code for a specific module.